test/runtests.jl

```diff
@@ -1,5 +1,7 @@
 using Test, AxisKeys, NamedDims
 using Statistics, OffsetArrays, Tables, UniqueVectors, LazyStack
+using ChainRulesTestUtils: test_rrule
```

Comment:
```suggestion
using ChainRulesTestUtils
```

Reply:
I guess it's just style, and since `test_rrule` is the main functionality of CRTU I guess it doesn't need to be told where it's from? Idk, don't feel strongly about this